src/docs/agenda.md

```diff
@@ -1,15 +1,17 @@
 # Agenda
 
-Under development: TBD
+!!! note
+    This section is currently under development. The information provided is subject to change.
 
 ## Tutorial Structure
 
-The tutorial will occur over half a day and focuses on introducing some common open science principles and their usage within education technology, providing an example on using the *Open Science Framework* to create a project, post content, and preregister studies, and using previous papers to apply the learned principles and any additional reproduction mitigations.
+The tutorial will occur over half a day and focuses on introducing some common open science principles and their usage within education technology, providing an example on using the [Open Science Framework][osf] to create a project, post content, and preregister studies, and using previous papers to apply the learned principles and any additional reproduction mitigations.
 
 1. We will provide a presentation on an overview of a few problems when conducting research. Using this as a baseline, we will introduce open science and its principles and how they can be used to nullify some of these issues and mitigate others. In addition, we will attempt to dispel some of the misconceptions of these principles.
```

Comment:
here i think we can say principles and practices, and misconceptions about adopting open science practices